cli/src/commands/config/mod.rs

```diff
@@ -17,8 +17,11 @@ mod get;
 mod list;
 mod path;
 mod set;
+mod unset;
 
 use tracing::instrument;
+use unset::cmd_config_unset;
+use unset::ConfigUnsetArgs;
```

Comment:
nit: `self::unset` for consistency (and clarity)

Reply:
Thanks, my LSP took care of that auto-import and I wasn't aware that it didn't do the right thing here.